app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] I think it would be better if you update the associations between models are set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Like for the **User** modal you should also specify the `foreign_key` for `:posts`, `:comments`, and `:comments`.
e.g.:
```ruby
    has_many :posts, foreign_key: 'author_id'
    .
    .
    .
```

Reply:
Changes made